quarto-materials/tiledb.scss

```diff
@@ -0,0 +1,26 @@
+/*
+Cloned from https://github.com/TileDB-Inc/tiledb-quarto-template
```

Comment:
I wonder if it would make sense to host this and other similar files centrally, and `@import` them in the downstream repositories?

Reply:
It would make sense to do that eventually, but that will complicate the build. It would be better to start now and get the centralization working later than to wait.

Reply:
@ihnorton definitely agreed! for the moment, these can be copied/synced, but centralized styles make a lot more sense.